src/Chainweb/BlockHeader.hs

```diff
@@ -285,35 +282,22 @@ powTarget
 powTarget p@(ParentHeader ph) as bct = case effectiveWindow ph of
     Nothing -> maxTarget
     Just w
-        -- A special case for starting a new devnet. Using maxtarget results in
-        -- a too high block production and consecutively orphans and network
-        -- congestion. The consequence are osciallations to take serval hundred
-        -- blocks before the system stabilizes. This code cools down initial
-        -- block production.
-        | ver == Development && _blockHeight ph == (genesisHeight ver cid + 1) -> HashTarget (maxBound `div` 100000)
```

Comment:
This hack is not needed any more, since now we define the initial target in the genesis blocks themself.